### PR TITLE
Retouch the look of reactions

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

[Retouch the look of reactions](https://github.com/webkom/lego-webapp/commit/c1e9841639f62a1d98fe8f56b2e8ed062d72e224) 

The new look resembles the new rounded and modern look of the webapp.

---
  
[Improve the reaction picker](https://github.com/webkom/lego-webapp/commit/266c7ab5b74e7f56eb0bc7a12eb849cd796a5eab) 

Uses the `<Card />` component, and has been given a more modern style.
Code was old and needed some work as well ..

# Result

**Before**
Screenshot 2023-02-01 at 22 43 05

**After**

The colors aren't any good on dark mode imo., but that will change soon with the incoming color fix. Kind of unnecessary to fix it for this PR.

image

**Before**

https://user-images.githubusercontent.com/69514187/216169557-719ec447-eadc-4cc3-9b15-26fecd60bf6e.mov

**After**

https://user-images.githubusercontent.com/69514187/216167578-42068707-cd6b-41be-91b4-7d8efaee2dad.mov

# Testing

- [x] I have thoroughly tested my changes.

I've tested it out on different viewports (mobile), dark-mode and different places (front-page and Overhørt-page).

---

Resolves ABA-228

### CODE REVIEW

File: app/components/Reactions/index.tsx [hunk 1]
```diff
@@ -101,16 +102,23 @@ class Reactions extends Component<Props, State> {
     } = this.props;
     const { reactionPickerOpen, addEmojiHovered } = this.state;
     return (
-      <div className={styles.reactionsContainer} ref={this.node}>
+      <div
+        className={styles.reactionsContainer}
+        ref={(node) => {
+          this.node = node;
+        }}
```
Comment (ivarnakken) [inline app/components/Reactions/index.tsx:109]: This fixes the bug in which the reactions picker closes when trying to click on the categories.

**Comment by SmithPeder** [comment]
After video looks a lot better 😍

> Small nitpick, but I feel like the "Active" background is a bit to strong

**Comment by ivarnakken** [comment]
> Small nitpick, but I feel like the "Active" background is a bit to strong

Yes, I sort of agree. It was difficult to pick a good "redish" color since our shades of red has not been retouched _yet_. That will be fixed by the end of the week.

**Comment by ingraso** [comment]
I've only looked at the design changes (photo and video) and I think it looks great! Much better with more color for the chosen reactions, and the rounded edges. The contrast between the number and the background color of the active reactions could be increased a bit, but this can be looked at during the retouching of the shades, as you're mentioning.

**Review by LudvigHz — APPROVED** [review]
LGTM

**Pull request merged**